packages/app/src/backup/services/backup-create.service.ts

```diff
@@ -15,20 +15,21 @@ import { isResponseSuccessWithData } from "App/core/helpers"
 import { BackupError } from "App/backup/constants"
 import { MetadataStore, MetadataKey } from "App/metadata"
 import { CreateDeviceBackup } from "App/backup/types"
+import { DeviceFileSystemService } from "App/device-file-system/services"
 
 // DEPRECATED
 import DeviceService from "App/__deprecated__/backend/device-service"
```

Comment:
`device-service.ts` will be refactored as part of the CP-1391? 🤔

Reply:
@dkarski It will totally destroyed, eliminated and removed from the project